### PR TITLE
Fix a few warnings

### PR DESCRIPTION
**Pull request by JoshDuMan**

### CODE REVIEW

**Comment by BowserSlug** [comment]
✅ This PR fixes 624 warnings!

File: .vscode/c_cpp_properties.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
             },
             "includePath": [
                 "${workspaceFolder}/include",
+                "${workspaceFolder}/include/script_api",
```
Comment (nanaian) [inline .vscode/c_cpp_properties.json:15]: Why?

**Review by nanaian — APPROVED** [review]
ty

File: ver/us/undefined_syms.txt [hunk 2]
```diff
@@ -1529,3 +1529,5 @@ D_800A15C4 = 0x800A15C4;
 D_800A15CC = 0x800A15CC;
 D_800B7EF0 = 0x800B7EF0;
 D_800B8DEC = 0x800B8DEC;
+dead_guMtxIdentF = 0x8006c660;
```
Comment (ethteck) [inline ver/us/undefined_syms.txt:1532]: uppercase address please

File: src/EF2680.c [hunk 3]
```diff
@@ -199,7 +199,7 @@ ApiStatus func_8024299C_EF4FAC(Evt* script, s32 isInitialCall) {
     Bytecode* args = script->ptrReadPos;
     EnemyTerritoryThing territory;
     EnemyTerritoryThing* territoryPtr = &territory;
-    NpcAISettings* aiSettings = evt_get_variable(script, *args);
+    NpcAISettings* aiSettings =(NpcAISettings*) evt_get_variable(script, *args);
```
Comment (ethteck) [inline src/EF2680.c:202]: spcae after =

**Review by ethteck — CHANGES_REQUESTED** [review]
great work! couple tiny things

**Pull request merged**